### PR TITLE
complete some rendering TODOs

### PR DESCRIPTION
**Pull request by Rexicon226**
The codebase is quite interesting, and I found it very fun to look through.

Follow-up question, should we end the global decls and error sets with a semi-colon?
Currently they render without one.
image

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -307,6 +307,34 @@
       domFnProto.classList.remove("hidden");
     }
 
+    function renderErrorSet(decl_index) {
+      renderNav(decl_index);
+      setViewSourceDecl(decl_index);
+
+      const members = namespaceMembers(decl_index, false);
+      if (members.length !== 0) {
+        resizeDomList(domListErrSets, members.length, '<li><a href="#"></a></li>');
```
Comment (Rexicon226) [inline lib/main.js]: I think this is the right html signature? Please correct me.

Comment (alichraghi) [reply lib/main.js]: `<a>` in `<li>`? yes that's correct

**Comment by andrewrk** [comment]
[here's a page to compare it to](https://ziglang.org/documentation/master/std/#A;std:fs.MAX_PATH_BYTES)

**Comment by Rexicon226** [comment]
> [here's a page to compare it to](https://ziglang.org/documentation/master/std/#A;std:fs.MAX_PATH_BYTES)

I see. So we should insert the docstring in the view as well. Can do now!

**Comment by Rexicon226** [comment]
@andrewrk Should the docstring be above or below the source? I personally think it should be above, but your choice.

**Comment by andrewrk** [comment]
Exercise taste! Show me your vision :)

**Comment by Rexicon226** [comment]
My computer clock once again decided that it is yesterday, but I pushed!
image

**Comment by andrewrk** [comment]
Can I see a screenshot of the error set page?

[comparison](https://ziglang.org/documentation/0.9.0/std/#std;event.Loop.Request.Msg.OpenAt.Error)

[note that current autodocs is regressed from this](https://ziglang.org/documentation/master/std/#A;std:posix.OpenError)

**Comment by Rexicon226** [comment]
It's currently in the middle of those two 😄 
I can instead of just rendering source, parse them like in the 0.9 docs. Give me a like 20 minutes to get home from school.
**this was all misguided**

**Comment by Rexicon226** [comment]
*Now* error sets work :)
Docstrings are a TODO, as I need to figure out how to best way to do them. Expect a follow up tomorrow. I think I've gotten over the Dunning-Kruger hill now, but time will tell.

image

**Comment by Rexicon226** [comment]
I've actually realized docstrings aren't that hard. Will push in a bit.

File: lib/index.html [hunk 2]
```diff
@@ -364,6 +364,11 @@ <h2>Error Sets</h2>
       <ul id="listErrSets">
       </ul>
     </div>
+    <div id="sectErrors" class="hidden">
+      <h2>Errors</h2>
+      <ul id="listErrors">
+      </ul>
+    </div>
```
Comment (andrewrk) [inline lib/index.html:371]: The section is already above, line 313

**Review by andrewrk — CHANGES_REQUESTED** [review]

**Comment by andrewrk** [comment]
previous error set implementation: https://github.com/andrewrk/autodoc/blob/a56d2c180084a065b12660d63cf0e88d502800d4/lib/main.js#L528-L560

**Comment by Rexicon226** [comment]
Sorry @andrewrk, I tabbed back to this GitHub page only once I had pushed. I didn't see you reviewing. I just cleaned it up a bit and added doc strings. I'm terrible at naming so please suggest fixes there as well :)

File: lib/index.html (hunk 2 above)
Comment (Rexicon226) [reply lib/index.html:371]: See it!

**Comment by andrewrk** [comment]
can you limit the PR to one enhancement? let's start with the const view, yeah?

**Comment by Rexicon226** [comment]
Gotcha. Apologies for being all over the place today :) I feel this PR has gotten very noisy and it's all my fault. Would you like me to close this and open two new ones for each feature? (global const and error set)?

**Pull request closed without merging**